server/command/main.go

```diff
@@ -25,10 +25,11 @@ func (s *service) handleMain(in *params) (*model.CommandResponse, error) {
 		"debug-install-builtin": s.executeDebugInstallBuiltinHello,
 		"debug-install-http":    s.executeDebugInstallHTTPHello,
 		"debug-install-aws":     s.executeDebugInstallAWSHello,
-		"provision":             s.executeProvision,
+		"provision-aws":         s.executeProvisionAWSApp,
+		"install-aws":           s.executeInstallAWSApp,
 		"info":                  s.executeInfo,
 		"list":                  s.executeList,
-		"install":               s.executeInstall,
+		"install":               s.executeInstallHTTPApp,
```

Comment:
3/5 that an the standard case should be install aws app. Should we even combine these two commands into a single `/install` command that either takes an `appID` of a listed AWS app `/install zendesk` or an URL `/install -url http://localhost:4000/manifest.json`?